src/mazzaroth.js

```diff
@@ -431,4 +439,97 @@ subCmd.action(function (val, options) {
   process.stdin.on('data', process.exit.bind(process, 0))
 })
 
+const deployCmd = program.command('deploy <val>')
+const deployCmdDescription = `
+Helper for deploying a contract to a mazzaroth network. Takes a json config file,
+a sample config file can be found at
+https://github.com/kochavalabs/full-contract-example/blob/master/example_deploy.json
```

Comment:
This link didn't work for me, looks like you renamed it to just deploy.json

Reply:
Fixed